charts/spire/values.yaml

```diff
@@ -108,6 +108,8 @@ spire-server:
   enabled: true
   ## @param spire-server.nameOverride Overrides the name of Spire server pods
   nameOverride: server
+  ## @param spire-server.deploymentType Run spire server as deployment/statefulset
```

Comment:
This feature should be marked in the comment as experimental explicitly so if folks are just looking at values, they will see it.

Reply:
Updated.